composer.json

```diff
@@ -16,7 +16,8 @@
     "require": {
         "php": "^5.5.0 || ^7.0",
         "psr/log": "^1.0",
-        "symfony/console": "^3.1"
+        "symfony/console": "^3.1",
+        "phpmyadmin/sql-parser": "^3.4"
```

Comment:
It might be good idea to raise this to `^4.1` to use current branch (it has been released today and has changed namespace to follow PSR-4).